app/src/main/java/com/stevesoltys/seedvault/restore/install/ApkRestore.kt

```diff
@@ -37,15 +38,22 @@ internal class ApkRestore(
 ) {
 
     private val pm = context.packageManager
+    private val restrictions = context.getSystemService(UserManager::class.java).userRestrictions
```

Comment:
Need to figure out how to mock this

Reply:
Maybe pass it as a constructor parameter via injection, so you can mock it here.